lab-taylor/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('runTests', () => {
+  gulp.src(__dirname + '/test/*.js')
+  .pipe(mocha());
+});
+
+gulp.task('runLint', () => {
+  return gulp.src(['**/*.js','!node_modules/**'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+
+gulp.task('default', ['runTests', 'runLint'], () => {
+});
+
+const watcher = gulp.watch(['**/*.js', '!node_modules/**'], ['runTests', 'runLint']);
```

Comment:
I suggest you declare and call your watch task differently, see my comment in Slack or ask me about it directly if you have any questions.